cli/src/main.rs

```diff
@@ -40,7 +40,7 @@ fn main() -> anyhow::Result<()> {
                 .long("go-package")
                 .help("Go package name")
                 .takes_value(true)
-                .required_if(ARG_TYPE, "go"),
+                .required(false),
```

Comment:
What would happen if the package name is specified neither in the flag nor in the config file?

Reply:
Similar to the other languages, it would come without a package name and this would error out in Go. I have updated it so theres a check to see if package name is given through the flags or `typeshare.toml`. If neither than error out.